assets/src/scss/editorOverrides.scss

```diff
@@ -165,14 +180,14 @@
 }
 
 .edit-post-visual-editor .editor-block-list__block-edit,
-.edit-post-visual-editor {
+.edit-post-visual-editor, .editor-styles-wrapper {
```

Comment:
I had to add this selector to make it apply our CSS rule, however it still ends up using the same serif font, probably because the font isn't loaded. In any case this can't hurt and is closer to what we intend.